cmd/pitr/storage/storage.go

```diff
@@ -129,15 +128,15 @@ func NewAzure(storageAccount, accessKey, endpoint, container, prefix string) (*A
 }
 
 func (a *Azure) GetObject(ctx context.Context, name string) (io.ReadCloser, error) {
-	resp, err := a.client.DownloadStream(ctx, a.container, url.QueryEscape(a.prefix+name), &azblob.DownloadStreamOptions{})
+	resp, err := a.client.DownloadStream(ctx, a.container, a.prefix+name, &azblob.DownloadStreamOptions{})
```

Comment:
We should do the same fix for other operators which use `github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v1.0.0`